### ISSUE TITLE
Error in user profile migration

### ISSUE THREAD

**Issue by dhylands** [mozilla-iot/gateway#769]
I tried running the gateway today (it's been a few weeks since the last time I ran it).
I pulled the latest source from the git repository and ran yarn to update the dependencies. I also updated my node to version v8.10.0. When I tried to launch using npm start, I get the following error:
```
Opening database: /home/dhylands/.mozilla-iot/config/db.sqlite3
Error: EISDIR: illegal operation on a directory, copyfile '/home/dhylands/Dropbox/moziot/gateway/build/addons/http-on-off-adapter' -> '/home/dhylands/.mozilla-iot/addons/http-on-off-adapter'
    at Object.fs.copyFileSync (fs.js:1918:11)
    at Object.renameFile (/home/dhylands/Dropbox/moziot/gateway/build/webpack:/src/user-profile.js:45:1)
    at Object.migrate (/home/dhylands/Dropbox/moziot/gateway/build/webpack:/src/user-profile.js:202:1)
    at Object../src/app.js (/home/dhylands/Dropbox/moziot/gateway/build/webpack:/src/app.js:14:1)
    at __webpack_require__ (/home/dhylands/Dropbox/moziot/gateway/build/webpack:/webpack/bootstrap:22:1)
    at ./src sync recursive.webpackEmptyContext.keys (/home/dhylands/Dropbox/moziot/gateway/build/webpack:/webpack/bootstrap:74:1)
    at Object.<anonymous> (/home/dhylands/Dropbox/moziot/gateway/build/gateway.js:79:10)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
```

**Comment by hobinjk** [comment]
Likely related to #760

**Comment by dhylands** [comment]
It seems that fs.copyFileSync requires a file as an argument and not a directory.

**Comment by dhylands** [comment]
It seems that fs-extra is already in node_modules, so we could add it to our dependencies and then do
```
fs = require('fs-extra')
fs.copy(src, dest)
```
where src can be a directory

**Comment by mrstegeman** [comment]
@dhylands it should be fixed with #770. Just a typo.

**Comment by dhylands** [comment]
Closing in favor of #770
